ui/src/assets/text/strings.ts

```diff
@@ -13,7 +13,7 @@ export const STRINGS = {
       agent_groups: 'Group your <a href="pages/fleet/agents">Agents</a> into <a href="pages/fleet/groups">Agent Groups</a> so that you can send your agents <a href="pages/datasets/policies">Policies</a>',
       policy: 'Create an <a href="pages/datasets/policies">Agent Policy</a> which will tell your agent how to analyze your data streams',
       sink: 'Setup a <a href="pages/sinks">Sink</a> to be able to send your agent metric output to a time series database for visualizing and alerting',
-      dataset: 'Finally, make a <a href="pages/datasets/list">Dataset</a> which will send your <a href="pages/datasets/policies">Policy</a> to the selected <a href="pages/fleet/groups">Agent Group</a> and the resulting metrics to the selected <a href="pages/sinks">Sink</a>',
+      dataset: 'Finally, make a Dataset which will send your <a href="pages/datasets/policies">Policy</a> to the selected <a href="pages/fleet/groups">Agent Group</a> and the resulting metrics to the selected <a href="pages/sinks">Sink</a>',
```

Comment:
As we discuss on last Demo, we are not supose to remove the dataset route from home page until the removal of the policies also remove all datasets linked. Lets wait #1068 to complete this task.